access-mgmt/README.adoc

```diff
@@ -0,0 +1,477 @@
++
+--
+The *Review existing permissions* section lists any permission settings in this instance that are already defined for all accounts in the organization and for the same account that you previously selected, if applicable. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Access* page.
+
+If you selected a permission entry that applies to a specific account, you can delete only permission entries that apply to individual accounts. If you selected a permission entry that applies to all accounts, you can delete only permission entries that apply to all accounts.
+--
+.. Under *Assign Permissions*, set the additional permissions for that account or all accounts for a resource type, such as a topic, and click *Add* to continue adding permissions for other resources as needed.
+.. Click *Save* to finish.
+
+* To delete existing account permissions in this instance, use the following options:
+
+ ** Select one or more permission entries, select the options icon (three vertical dots) at the top of the table, and click *Delete selected permissions*.
+ ** For the account that you want to delete, select the options icon (three vertical dots) for that entry and click *Delete*.
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://kafka.apache.org/documentation/#security_authz[Authorization and ACLs^] in Kafka documentation
+
+[id="ref-example-access-scenarios_{context}"]
+== Example account access scenarios in {product}
+
+[role="_abstract"]
+The following example Access Control Lists (ACLs) illustrate common scenarios for managing the level of access for user accounts or service accounts in {product}. Use these examples as a guide for your own ACLs.
+
+Access for a new service account in a Kafka instance::
++
+--
+I’ve created a new service account and I want to allow it to create and delete topics in the instance, to produce and consume messages in any topic in the instance, and to use any consumer group and any producer.
+
+.Example ACL permissions
+[cols="25%,22%,23%,15%,15%"]
+|===
+h|Account
+h|Resource type
+h|Resource identifier
+h|Access type
+h|Operation
+
+|`srvc-acct-1a2b3c4d-...`
+|`Topic`
+|`Is` = `*`
+|`Allow`
+|`All`
+
+|`srvc-acct-1a2b3c4d-...`
+|`Consumer group`
+|`Is` = `*`
+|`Allow`
+|`Read`
+
+|`srvc-acct-1a2b3c4d-...`
+|`Transactional ID`
+|`Is` = `*`
+|`Allow`
+|`All`
+|===
+--
+
+Access for all accounts in a Kafka instance::
++
+--
+I want this Kafka instance to be fully accessible to all accounts in the organization. I want any user to be able to read all topics, write to all topics, use any consumer group, and use any producer.
+
+.Example ACL permissions
+[cols="25%,22%,23%,15%,15%"]
+|===
+h|Account
+h|Resource
+h|Resource identifier
+h|Access type
+h|Operations
+
+|`All accounts`
+|`Topic`
+|`Is` = `*`
+|`Allow`
+|`All`
+
+|`All accounts`
+|`Consumer group`
+|`Is` = `*`
+|`Allow`
+|`All`
```

Comment:
This is likely some ignorance on my part, but I noticed that the operation for a _service account_ to use any consumer group is `Read`, while for _human user accounts_ and `All accounts` it is `All`. I presume there's a good reason for this, and it's not a mistake? :)